examples/tree-view/containers/Node.js

```diff
@@ -23,16 +24,23 @@ class Node extends Component {
     addChild(id, childId)
   }
 
+  handleRemoveChildClick(childId, e) {
+    e.preventDefault();
+
+    const { removeChild, id } = this.props
+    removeChild(id, childId)
+  }
+
   renderChild(childId) {
     return (
       <li key={childId}>
-        <ConnectedNode id={childId} />
+        <ConnectedNode id={childId} handleRemoveChildClick={this.handleRemoveChildClick.bind(this, childId)} />
       </li>
     )
   }
 
   render() {
-    const { counter, childIds } = this.props
+    const { counter, childIds, handleRemoveChildClick } = this.props
```

Comment:
This would rather become a method on the component called `handleRemoveClick` that would read its own ID from `props` and pass it down.